tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,105 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
+    Change The Sort and Check With The List    ${list_of_ids_of_favourite}    duration
+
+    Remove The Range Of Tiles From Favourite    ${list_of_tile}    27    48
+
 *** Keywords ***
+Verify Star Icons Are Clickable
+    [Documentation]    Verifies that star icons in the resources page are clickable
+    [Arguments]    ${list_of_ids}
+    FOR    ${id}    IN    @{list_of_ids}
+         Set Item As Favourite    ${id}
+         Remove Item From Favourite    ${id}
+    END
+
+List Should Be Equal
+    [Documentation]    Checks that lists are equal or not,
+    ...                end indexes are exclusive in range
+    [Arguments]    ${list1}    ${start_index1}    ${end_index1}
+    ...            ${list2}    ${start_index2}    ${end_index2}    ${step_list2}=${1}
+    FOR    ${index1}  IN RANGE    ${start_index1}    ${end_index1}    ${step_list2}
+        Should Be Equal    ${list1}[${index1}]    ${list2}[${start_index2}]
+        ${start_index2} =    Evaluate    ${start_index2} + ${1}
+    END
+
+Get List Of Ids Of Tiles
+    [Documentation]    Returns the list of ids of tiles present in resources page
+    ${list_of_ids}=    Get List Of Atrributes
+    ...    xpath=//article[@class="pf-c-card pf-m-selectable odh-card odh-tourable-card"]    attribute=id
+    [Return]    ${list_of_ids}
+
+Set Item As Favourite
+    [Documentation]    Add the tiles in favourite
+    [Arguments]    ${id}
+    ${not_clicked} =    Get Element Attribute    //*[@id="${id}"]/div[1]/span    class
+    Should Be Equal    ${not_clicked}    odh-dashboard__favorite
+    Click Element    //*[@id="${id}"]/div[1]/span
+
+Remove Item From Favourite
+    [Documentation]    Removes the tiles from favourite
+    [Arguments]    ${id}
+    ${clicked} =    Get Element Attribute    //*[@id="${id}"]/div[1]/span    class
+    Should Be Equal    ${clicked}    odh-dashboard__favorite m-is-favorite
+    Click Element    //*[@id="${id}"]/div[1]/span
+
+Add The Range Of Tiles In Favourite
+    [Documentation]    Adds the range of tiles in favourite
+    [Arguments]    ${list_of_ids}    ${start_index}    ${end_index}
+    FOR  ${index}  IN RANGE    ${start_index}    ${end_index}
+         Set Item As Favourite    ${list_of_ids}[${index}]
+    END
+
+Remove The Range Of Tiles From Favourite
+    [Documentation]    Adds the range of tiles in favourite
+    [Arguments]    ${list_of_ids}    ${start_index}    ${end_index}
+    FOR  ${index}  IN RANGE    ${start_index}    ${end_index}
+         Remove Item From Favourite    ${list_of_ids}[${index}]
+    END
+
+Change The Sort and Check With The List
+    [Arguments]    ${list_of_ids_of_favourite}    ${sort_type}
+    Change The Sort    ${sort_type}
+    ${new_list_of_tile} =    Get List Of Ids Of Tiles
+    ${new_list_of_tile} =    Get Slice From List    ${new_list_of_tile}    0    21
+    Lists Should Be Equal    ${new_list_of_tile}    ${list_of_ids_of_favourite}    ignore_order=${True}
+
+Change The Sort
```

Comment:
Could be renamed to `Sort Resources By` and be placed in ODHDashboard.robot?  Other keywords in this test could be useful too to be shared

Reply:
Done